packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -271,4 +235,32 @@ export default class EthereumAnchorService extends AnchorService {
         return providers.getDefaultProvider(ethNetwork.network)
     }
 
+    private processRemoteResponse(cidDoc: CidDoc, json: any): boolean {
```

Comment:
Can you add a comment for this method?  At the very least it should clarify what the return value represents

Reply:
The replacement for this is coming in the next batch: https://github.com/ceramicnetwork/js-ceramic/pull/860/files#diff-7920408ff457031025a51c6c64241f297dc91febb1a810e724475548c154c781R251